### ISSUE TITLE
Inconsistent spacing when form fields are inside a details element

### ISSUE THREAD

**Issue by gosko** [w3c/w3c-website#686]
On the [Mailing list archive search page](https://www.w3.org/Search/Mail/Public/search) some options are hidden inside a `details` element:

```html

<details>
<summary>Advanced search options</summary>

<div class="field">
[...]
```

These options are displayed quite close together, with no space between them:
<img width="700" alt="Screenshot 2024-11-05 at 11 27 30 AM" src="https://github.com/user-attachments/assets/589131fe-c1bb-4a7b-aeff-f3d1f36e6be2">

unlike the options at the top, outside the `details` element:
<img width="700" alt="Screenshot 2024-11-05 at 11 27 11 AM" src="https://github.com/user-attachments/assets/3302994a-7cdf-4380-82ed-6d1466391bb4">

It would be good for the items inside the `details` element to have the same spacing as those outside it.

(Also if there is a better way to handle this form I am open to suggestions. The [previous version of this page](http://web.archive.org/web/20240609141239/https://www.w3.org/Search/Mail/Public/) had separate pages for Basic Search and Advanced Search)

**Comment by jean-gui** [comment]
This seems to be fixed with https://github.com/w3c/w3c-website-templates-bundle/pull/176/. @gosko Can you confirm? You can test using https://www-dev.w3.org/assets/website-2021-dev/pr-176/styles/core.css?ver=1.5?

**Comment by gosko** [comment]
Yes, this seems fixed using the latest styles on www-dev, thanks!
